File: src/test/java/net/rentalhost/plugins/php/hammer/inspections/codeStyle/NullableTypeRightmostInspectionTestCase.java
package net.rentalhost.plugins.php.hammer.inspections.codeStyle;

import net.rentalhost.plugins.services.TestCase;

public class NullableTypeRightmostInspectionTestCase
    extends TestCase {
    public void testAll() {
        testInspection(NullableTypeRightmostInspection.class);
    }
}

File: src/test/java/net/rentalhost/plugins/php/hammer/inspections/codeStyle/NullableTypeRightmostInspectionTestCase.kt
package net.rentalhost.plugins.php.hammer.inspections.codeStyle

import net.rentalhost.plugins.services.TestCase

class NullableTypeRightmostInspectionTestCase: TestCase() {
    fun testAll() = testInspection(NullableTypeRightmostInspection::class.java)
}
